pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -695,6 +697,7 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 	if err != nil {
 		return errors.Wrap(err, "k.Transform failed")
 	}
+	selector := "io.kompose.service"
```

Comment:
@cdrage you are right. I was thinking about it, but forgot to commend on it :-)

This should be even somewhere else (maybe const in transformer pkg?), so [ConfigLabels](https://github.com/kubernetes-incubator/kompose/blob/5cb598fa5bed08ffe52b4a02fd299fbf3de373e3/pkg/transformer/utils.go#L118) function can use it, and we don't have same magic string in two places.